src/pages/@owner/food-truck-form/FoodTruckForm.tsx

```
@@ -0,0 +1,110 @@
+    </FormProvider>
+  );
+}
+
+// 제출 버튼 컴포넌트
+function SubmitButton() {
+  const {
+    handleSubmit,
+    formState: { isValid },
+  } = useFormContext();
+
+  const onSubmit = async (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)